README.md

```diff
@@ -1,2 +1,134 @@
-# mapstore2-longitudinal-profile
-Repository for the longitudinal profile plugin for MapStore2 for geOrchestra
+# Longitudinal profile plugin for geOrchestra MapStore2 viewer
+
+# General
+
+This repository is a [MapStore Extension](https://mapstore.readthedocs.io/en/latest/developer-guide/extensions/) repository to build longitudinal profile plugin for the [MapStore2 geOrchestra project](https://github.com/georchestra/mapstore2-georchestra).
+
+It can be used also as a template to create new extensions for geOrchestra.
+
+## Quick Start
+
+Clone the repository with the --recursive option to automatically clone submodules.
+
+`git clone --recursive https://github.com/georchestra/mapstore2-longitudinal-profile`
+
+Install NodeJS >= 12.16.1 , if needed, from [here](https://nodejs.org/en/download/releases/).
+
+You can start the development application locally:
+
+`npm install`
+
+`npm start`
+
+The application runs at `http://localhost:8081` afterwards. You will see, opening a map, the sample plugin on top of the map.
+
+## Plugin configuration
+
+Longitudinal profile plugin allows the user to build profile based on the drawn, selected or imported geometry data corresponding to that geometry, obtained from WPS backend.
```

Comment:
Given a selected digital elevation model, the Longitudinal profile plugin allows the user to generate a longitudinal profile based on the drawn, selected or imported geometry. The longitudinal profile is calculated by a [WPS process](https://gitlab.agglo-lepuyenvelay.fr/jusabatier/profil-en-long-wps) and retrieved from it.